packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -44,14 +51,14 @@ const double _naturalPixelRatio = 1.0;
 /// a default marker icon.
 /// Use the [BitmapDescriptor.defaultMarkerWithHue] to create a
 /// [BitmapDescriptor] for a default marker icon with a hue value.
-class BitmapDescriptor {
-  const BitmapDescriptor._(this._json);
+abstract class BitmapDescriptor {
```

Comment:
I thought this was a breaking change but a spot check of the public usages in github show I am incorrect.

Reply:
The only ways to construct a `BitmapDescriptor` without constructing one of its subclasses was either through this private `_` factory, which is removed here, or the below `fromJson` public factory, which this PR converts to a static method, so unless we are expecting public usages of this package to add new subclasses of `BitmapDescriptor`, I believe that neither making it abstract nor removing the `_` factory should impact any public uses.